<!DOCTYPE html>
<html>
<head>
	<title>Reference</title>
	<meta charset="utf-8">
	<link rel="stylesheet" type="text/css" href="public/css/base.css">
	<link rel="stylesheet" type="text/css" href="public/css/project.css">
</head>
<body>
	<nav>
		<div id="name"><p>CampusMicrobiome</p></div>
		<div class="item">
			<ul>
				<li><a href="home">Home</a></li>
				<li class="dropdown">
					<a class="drop_btn" href="">Project</a>
					<div class="drop_cont">
						<dl>
							<dd><a href="p_intro">Introduction</a></dd>
							<dd><a href="p_res">Results</a></dd>
							<dd><a href="p_meth">Methods</a></dd>
							<dd><a href="p_ref">References</a></dd>
						</dl>					
					</div>
				</li>
				<li><a href="outreach">Outeach</a></li>
				<li><a href="about">About</a></li>
			</ul>
		</div>
	</nav>

	<div id="ref">
		<h3>Reference</h3>
		<p>1.	Falkowski PG, Fenchel T, Delong EF. The microbial engines that drive Earth's biogeochemical cycles. Science. 2008;320:1034-.</p>
		<p>2.	Bossio DA, Girvan MS, Verchot L, Bullimore J, Borelli T, Albrecht A, Scow KM, Ball AS, Pretty JN, Osborn AM. Soil microbial community response to land use change in an agricultural landscape of western Kenya. Microb Ecol. 2005;49:50-62.</p>
		<p>3.	Barberán A, Bates ST, Casamayor EO, Fierer N. Using network analysis to explore co-occurrence patterns in soil microbial communities. Isme Journal. 2011;6:343-351.</p>
		<p>4.	Wall DH. Sustaining biodiversity and ecosystem services in soils and sediments. Island Press; 2004.</p>
		<p>5.	Fuhrman JA, Cram JA, Needham DM. Marine microbial community dynamics and their ecological interpretation. Nature Reviews Microbiology. 2015;13:133-146.</p>
		<p>6.	Wang Y, Sheng HF, He Y, Wu JY, Jiang YX, Tam NF, Zhou HW. Comparison of the levels of bacterial diversity in freshwater, intertidal wetland, and marine sediments by using millions of illumina tags. Appl Environ Microbiol. 2012;78:8264-8271.</p>
		<p>7.	Li J, Zhang J, Liu L, Fan Y, Li L, Yang Y, Lu Z, Zhang X. Annual periodicity in planktonic bacterial and archaeal community composition of eutrophic Lake Taihu. Scientific Reports. 2015;5:15488.</p>
		<p>8.	Han M, Gong Y, Zhou C, Zhang J, Wang Z, Kang N. Comparison and Interpretation of Taxonomical Structure of Bacterial Communities in Two Types of Lakes on Yun-Gui plateau of China. Scientific Reports. 2016;6:30616.</p>
		<p>9.	Ward L, Taylor MW, Power JF, Scott BJ, Mcdonald IR, Stott MB. Microbial community dynamics in Inferno Crater Lake, a thermally fluctuating geothermal spring. Isme Journal. 2017.</p>
		<p>10.	Concepción ED, Moretti M, Altermatt F, Nobis MP, Obrist MK. Impacts of urbanisation on biodiversity: the role of species mobility, degree of specialisation and spatial scale. Oikos. 2015;124:1571-1582.</p>
		<p>11.	Reese AT, Savage A, Youngsteadt E, McGuire KL, Koling A, Watkins O, Frank SD, Dunn RR. Urban stress is associated with variation in microbial species composition-but not richness-in Manhattan. ISME J. 2016;10:751-760.</p>
		<p>12.	Albert Barberán JL, Jonathan W. Leff, Katherine S. Pollard, Holly L. Menninger ,Robert R. Dunn , and Noah Fierer. Continental-scale distributions of dust-associated bacteria and fungi.pdf>. PNAS. 2015;112:5756-5761.</p>
		<p>13.	de Voogd NJ, Cleary DF, Polonia AR, Gomes NC. Bacterial community composition and predicted functional ecology of sponges, sediment and seawater from the thousand islands reef complex, West Java, Indonesia. FEMS Microbiol Ecol. 2015;91.</p>
		<p>14.	Pesant S, Not F, Picheral M, Kandels-Lewis S, Le Bescot N, Gorsky G, Iudicone D, Karsenti E, Speich S, Trouble R, et al. Open science resources for the discovery and analysis of Tara Oceans data. Sci Data. 2015;2:150023.</p>
		<p>15.	Shinichi Sunagawa LPC, Samuel Chaffron  Jens Roat Kultima, Karine Labadie, 5 Guillem Salazar, Bardya Djahanschiri, Georg Zeller, Daniel R. Mende,  Adriana Alberti, Francisco M. Cornejo-Castillo, Paul I. Costea, Corinne Cruaud, Francesco d'Ovidio, Stefan Engelen, Isabel Ferrera,  Josep M. Gasol, Lionel Guidi, Falk Hildebrand, Florian Kokoszka,  Cyrille Lepoivre, Gipsi Lima-Mendez,  Julie Poulain,  Bonnie T. Poulos,  Marta Royo-Llonch, Hugo Sarmento,  Sara Vieira-Silva,  Céline Dimier,  Marc Picheral, 8,9, Sarah Searson, 9 Stefanie Kandels-Lewis, 1,17 Tara Oceans coordinators‡ Chris Bowler, 10, Colomban de Vargas, 16 Gabriel Gorsky, 8,9 Nigel Grimsley, 18,19 Pascal Hingamp, 12, Daniele Iudicone OJ, 5,21,22 Fabrice Not, 15,16 Hiroyuki Ogata, 23, Stephane Pesant, 25 Sabrina Speich, 26,27 Lars Stemmann, 8,9 Matthew B. Sullivan, 13§, Jean Weissenbach, 21,22 Patrick Wincker, 5,21,22 Eric Karsenti, 10,17 † Jeroen Raes, 2,3,4 †, Silvia G. Acinas PB. Structure and function of the global ocean microbiome. Science. 2015;348.</p>
		<p>16.	Loman NJ, Pallen MJ. Twenty years of bacterial genome sequencing. Nat Rev Microbiol. 2015;13:787-794.</p>
		<p>17.	Ilkka Hanski  LvH, Nanna Fyhrquist, Kaisa Koskinen, Kaisa Torppa, Tiina Laatikainen , Piia Karisola,, Petri Auvinen LP, Mika J. Mäkelä, Erkki Vartiainen, Timo U. Kosunen, Harri Alenius, and Tari Haahtela. Environmental biodiversity, human microbiota, and allergy are interrelated.pdf. PNAS. 2012;109:8334-8339.</p>
		<p>18.	Fujimura KE, Demoor T, Rauch M, Faruqi AA, Jang S, Johnson CC, Boushey HA, Zoratti E, Ownby D, Lukacs NW. House dust exposure mediates gut microbiome Lactobacillus enrichment and airway immune defense against allergens and virus infection. Proceedings of the National Academy of Sciences of the United States of America. 2014;111:805-810.</p>
		<p>19.	Lax S, Smith DP, Hampton-Marcell J, Owens SM, Handley KM, Scott NM. Longitudinal analysis of microbial interaction between humans and the indoor environment. Science. 2014;345.</p>
		<p>20.	Zifcakova L, Vetrovsky T, Lombard V, Henrissat B, Howe A, Baldrian P. Feed in summer, rest in winter: microbial carbon utilization in forest topsoil. Microbiome. 2017;5:122.</p>
		<p>21.	Tong X, Leung MHY, Wilkins D, Lee PKH. City-scale distribution and dispersal routes of mycobiome in residences. Microbiome. 2017;5:131.</p>
		<p>22.	Chaffron S, Rehrauer H, Pernthaler J, Mering CV. A global network of coexisting microbes from environmental and whole-genome sequence data. Genome Research. 2010;20:947-959.</p>
		<p>23.	Bodelier PL. Toward understanding, managing, and protecting microbial ecosystems. Frontiers in Microbiology. 2011;2:80.</p>
		<p>24.	Sumampouw OJ. Bacteria as Indicators of Environmental Pollution: Review. International Journal of Ecosystem. 2014;4(6):251-258.</p>
		<p>25.	Ashok V, Gupta T, Dubey S, Jat R. Personal exposure measurement of students to various microenvironments inside and outside the college campus. Environmental Monitoring and Assessment. 2013;186:735-750.</p>
		<p>26.	Cheng X, Chen X, Su X, Zhao H, Han M, Bo C, Xu J, Bai H, Ning K. DNA extraction protocol for biological ingredient analysis of Liuwei Dihuang Wan. Genomics Proteomics Bioinformatics. 2014;12:137-143.</p>
		<p>27.	Cheng X, Su X, Chen X, Zhao H, Bo C, Xu J, Bai H, Ning K. Biological ingredient analysis of traditional Chinese medicine preparation based on high-throughput sequencing: the story for Liuwei Dihuang Wan. Scientific Reports. 2014;4:5147-5147.</p>
		<p>28.	Porebski S, Bailey LG, Baum BR. Modification of a CTAB DNA extraction protocol for plants containing high polysaccharide and polyphenol components. Plant Molecular Biology Reporter. 1997;15:8-15.</p>
		<p>29.	Schloss PD, Westcott SL, Ryabin T, Hall JR, Hartmann M, Hollister EB, Lesniewski RA, Oakley BB, Parks DH, Robinson CJ, et al. Introducing mothur: open-source, platform-independent, community-supported software for describing and comparing microbial communities. Appl Environ Microbiol. 2009;75:7537-7541.</p>
		<p>30.	Quast C, Pruesse E, Yilmaz P, Gerken J, Schweer T, Yarza P, Peplies J, Glöckner FO. The SILVA ribosomal RNA gene database project: improved data processing and web-based tools. Nucleic Acids Research. 2013;41:D590.</p>
		<p>31.	Caporaso JG, Bittinger K, Bushman FD, Desantis TZ, Andersen GL, Knight R. PyNAST: a flexible tool for aligning sequences to a template alignment. Bioinformatics. 2010;26:266.</p>
		<p>32.	Caporaso JG, Kuczynski J, Stombaugh J, Bittinger K, Bushman FD, Costello EK, Fierer N, Peña AG, Goodrich JK, Gordon JI. QIIME allows analysis of high-throughput community sequencing data. Nature Methods. 2010;7:335-336.</p>
		<p>33.	Desantis TZ, Hugenholtz P, Larsen N, Rojas M, Brodie EL, Keller K, Huber T, Dalevi D, Hu P, Andersen GL. Greengenes: Chimera-checked 16S rRNA gene database and workbenchcompatible in ARB. Applied & Environmental Microbiology. 2006;72:5069-5072.</p>
		<p>34.	Solow A. Measuring Biological Diversity. Environmental and Ecological Statistics. 1994;30:814–815.</p>
		<p>35.	Lozupone C, Knight R. UniFrac: a new phylogenetic method for comparing microbial communities. Appl Environ Microbiol. 2005;71:8228-8235.</p>
		<p>36.	Vázquezbaeza Y, Pirrung M, Gonzalez A, Knight R. EMPeror: a tool for visualizing high-throughput microbial community data. GigaScience. 2013;2:16.</p>
		<p>37.	Segata N, Izard J, Waldron L, Gevers D, Miropolsky L, Garrett WS, Huttenhower C. Metagenomic biomarker discovery and explanation. Genome Biol. 2011;12:R60.</p>
		<p>38.	Zhou J, Deng Y, Luo F, He Z, Tu Q, Zhi X. Functional Molecular Ecological Networks. Mbio. 2010;1:1592-1601.</p>
		<p>39.	Shannon P, Markiel A, Ozier O, Baliga NS, Wang JT, Ramage D, Amin N, Schwikowski B, Ideker T. Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks. Genome Research. 2003;13::2498-2504.</p>
		<p>40.	Bader GD, Hogue CW. An automated method for finding molecular complexes in large protein interaction networks. BMC Bioinformatics. 2003;4:2.</p>
		<p>41.	Weiss S, Treuren WV, Lozupone C, Faust K, Friedman J, Ye D, Li CX, Xu ZZ, Ursell L, Alm EJ. Correlation detection strategies in microbial data sets vary widely in sensitivity and precision. Isme Journal. 2016;10:1669.</p>
	</div>

	<footer>
		<div id="f3">
			<strong>Contact Us</strong>
			<p id="f0">Huazhong University of Science & Technology</p>
			<div id="f1">
				<dl>
					<dt><img src="public/webpic12.13/phone.png"></dt>
					<dd><p>+86 027 87793041</p></dd>
				</dl>
				<dl>	
					<dt><img src="public/webpic12.13/email.png"></dt>
					<dd><p>ningkang@hust.edu.cn</p></dd>
				</dl>
				<dl>
					<dt><img src="public/webpic12.13/map.png"></dt>
					<dd><p>Wuhan,Luoyu Road</p></dd>
				</dl>
			</div>
		</div>
		
		<p id="f2">The School of Life Science and Technology, Huazhong University of Science & Technology © 2017 all rights reserved.</p>
	</footer>
</body>
</html>